client/ayon_core/tools/workfiles/widgets/save_as_dialog.py

```diff
@@ -143,6 +143,11 @@ def __init__(self, controller, parent):
         version_layout.addWidget(version_input)
         version_layout.addWidget(last_version_check)
 
+        # Artist note widget
+        artist_note_input = QtWidgets.QPlainTextEdit(inputs_widget)
```

Comment:
Requires import `from ayon_core.tools.utils import PlaceholderPlainTextEdit`
```suggestion
        artist_note_input = PlaceholderPlainTextEdit(inputs_widget)
```